SB2018/2018-03-23.md

```diff
@@ -0,0 +1,14 @@
+---
+to: All Teams
+subject: Media Consent Forms
+---
+
+Hi all,
+
+In order for anyone to attend the competition, a Media Consent Form must be filled out. If students are over 16, they can fill out the forms themselves; however, if students are under 16, the form must be signed by their parent/guardian.
+
+These forms must be filled and sent to us to get a ticket for the competition, and to ultimately gain entry.
+
+To submit a form, just send me an email with an attached PDF of a scan of the the filled out forms. We may reply with any errors we see.
+
+Please try to get the tickets to us as soon as possible, as we'll be extremely busy organising the competition itself in the week before. We will be processing forms up to the 17th of April, if forms aren't filled out by then, you can bring them to the competition to be checked on the door. Any additional guests can sign a form on the door as long as they're 16 or over.
```

Comment:
> Please try to get the tickets to us as soon as possible, as we'll be e

Should this be MCF not tickets?

Reply:
I could've sworn I fixed that... fixed in F397374